pkg/utils/controller/binding_resolver.go

```diff
@@ -67,7 +68,19 @@ func ListBindingsFromKey(ctx context.Context, c client.Reader, placementKey type
 		return nil, NewAPIServerError(false, err)
 	}
 
-	return bindingList.GetBindingObjs(), nil
+	bindingObjs := bindingList.GetBindingObjs()
+
+	// Sort the list of bindings.
+	//
+	// This is needed to ensure deterministic decision output from the scheduler.
+	sort.Slice(bindingObjs, func(i, j int) bool {
```

Comment:
just curious, why doesn't the client-go or api-server not sorting the list of bindings when they return them?